### PR TITLE
adjusts menu and section margins

### PR DESCRIPTION
**Pull request by caseyyee**
image

### CODE REVIEW

File: themes/aframe/source/css/_settings.styl [hunk 1]
```diff
@@ -37,3 +37,6 @@ $copy-link-hover = #35c5e9  // hsl(192, 80%, 56%)
 
 // Miscellaneous
 $radius = 2px
+
+// Margins
+$margin = 15px
```
Comment (ngokevin) [inline themes/aframe/source/css/_settings.styl]: Could be more specific on the type of margin, and perhaps contain to the `home.styl` so we can clean up later.

File: themes/aframe/source/css/home.styl [hunk 2]
```diff
@@ -26,17 +26,18 @@ ul
 .home__faq
   display flex
   flex-wrap wrap
-  margin-bottom 30px
   width 100%
   > h2
-    margin-bottom 15px
+    margin-top 30px
+    padding $margin
```
Comment (ngokevin) [inline themes/aframe/source/css/home.styl]: padding - margin bit weird. Maybe needs a different name.

Comment (caseyyee) [inline themes/aframe/source/css/home.styl]: good call.

File: themes/aframe/source/css/_settings.styl [hunk 3]
```diff
@@ -37,3 +37,6 @@ $copy-link-hover = #35c5e9  // hsl(192, 80%, 56%)
 
 // Miscellaneous
 $radius = 2px
+
+// Layout paddings
+$padding-med = 15px
```
Comment (caseyyee) [inline themes/aframe/source/css/_settings.styl]: @ngokevin yeah, will be adding a bunch of these, so this should do for now?

File: themes/aframe/source/css/home.styl [hunk 4]
```diff
@@ -50,8 +51,9 @@ ul
 
 .home__nav
   display flex
-  li
-    margin-right 10px
+  margin 30px $padding-med $padding-med 0
```
Comment (ngokevin) [inline themes/aframe/source/css/home.styl]: now it says margin: padding

Comment (caseyyee) [inline themes/aframe/source/css/home.styl]: Hehe.  I don't want to use another name for white spacing.  but $spacing-med  might be better?

File: themes/aframe/source/css/_settings.styl (hunk 3 above)
Comment (ngokevin) [inline themes/aframe/source/css/_settings.styl]: yeah, might want separate ones for margin/padding, and we can tweak names later

File: themes/aframe/source/css/home.styl (hunk 4 above)
Comment (ngokevin) [inline themes/aframe/source/css/home.styl]: sounds good. or `$section-spacing` if that makes sense to be semantic

Comment (caseyyee) [inline themes/aframe/source/css/home.styl]: I use it in a bunch of places.  I'll go with _spacing_ for now and we can figure out better naming a bit later.  Thanks!

**Pull request merged**